src/Domain/FundingWithdrawal.php

```diff
@@ -10,6 +10,8 @@
  * @ORM\Entity(repositoryClass="FundingWithdrawalRepository")
  * @ORM\HasLifecycleCallbacks
  * @ORM\Table
+ *
+ * @psalm-suppress PropertyNotSetInConstructor Not requiring all props on construct for now.
```

Comment:
yep, or maybe better add the new Psalm errors to the baseline? We don't require the props defined in old code to be on construct, but IMHO we should require any future props without default values to be set in the constructor.